.github/workflows/deploy.yml

```diff
@@ -42,8 +42,8 @@ jobs:
       - name: Deploy with kamal
         run: |
           kamal lock release
-          kamal accessory boot yosys2digitaljs-server
-          kamal traefik boot
+          kamal accessory boot yosys2digitaljs-server || true
```

Comment:
Move this to separate PR

Reply:
Done @tachyons, first need to merge this then the other one https://github.com/CircuitVerse/CircuitVerse/pull/4001